tests/pallas/pallas_test.py

```diff
@@ -458,6 +458,29 @@ def softmax_kernel(x_ref, y_ref):
 
     np.testing.assert_allclose(softmax_kernel(x), jax.nn.softmax(x), atol=1e-7)
 
+  def test_block_spec_with_padding(self):
+    self.INTERPRET = True
```

Comment:
Maybe add it to the `PallasCallInterpreterTest` below?

Reply:
This was left over from my debugging. Removed.